integrations/extensions/starter-kits/language-model-complex-informational-tasks/README.md

```diff
@@ -69,3 +69,58 @@ Below is a list of the session variables used in this example. They are automati
+| Purchase a Plan         | A conventional action that guides the user through steps with multiple choice options for purchasing or upgrading their TV and Internet plans. This action is included in this kit to demonstrate how language model actions and conventional actions can co-exist in the same assistant. |
+
+#### Session variables
+
+Below is a list of the session variables used in this example. They are automatically set with defaults in the sample [complex-info-palm-actions.json](./complex-info-palm-actions.json), so you do not need to set them yourself unless you want to make changes.
+
+- `internet_plan_prompt`: The prompt for the PaLM model on the topic of internet plans. Specifically, this prompt is "You are an agent for a cable provider that helps customers determine which internet plan is best for them. Limit your response to 30 words and only discuss which plans are best for that customer unless prompted otherwise. The three plans are 1) The 100 Mbps plan which cost $30; 2) The 500 Mbps which cost $40; 3) The 1 Gbps plan which cost $50"
+- `tv_plan_prompt`: The prompt for the PaLM model on the topic of TV plans. It includes instructions and data for the model to use when generating an answer.
+- `model_prompt`: This is the prompt for PaLM model which is a concatinated string of internet_plan_prompt or tv_plan_prompt and the input text. This entire string is Context and instructions, relevent text as a knowledgebase and a user query which is fed to the LLM model.
+- `verbose`: When set to `True` prints debugging information. Default is `False`. In this starter kit, when `verbose` is `True`, a step in the "Generate Answer" action will print out the prompt that is being sent to the model.
+- `model_response`: The text generated by the model in response to the `model_prompt`.
+
+### Example usage
+
+#### An example of how to use the `Internet Plan Questions` action for this starter kit:
+
+<img src="./assets/complex-info-palm-internet-plan-example.png" width="300"/>
+
+## Use at your own risk
+
+The prompts in this kit provide specific details for the models to pull information from, and often the models are quite good at focusing on those details. However, depending on the question, the details of the prompt, and the model used, the models can also provide additional information that is not in the prompt and may not always be what you want your assistant to say. We strongly recommend that anyone using this approach do a great deal of testing, and think carefully before deploying something like this. It can allow you to get some impressive results very quickly and easily, but it comes with some significant risk of producing misinformation in some cases. Careful revision of the prompts can reduce that risk, but probably will not eliminate it completely.
```

Comment:
Just link to this section above rather than repeating it. Since you have additional text to add for PaLM, you can do something like Refer to [this section for risks of using these LLM models](link-to-section). 

Also for this PaLM example, ...